config/iam-components.yaml

```diff
@@ -0,0 +1,1152 @@
+postgresql:
```

Comment:
In general about the whole values file:
Can this be reduced to a minimal example? I suggest to leave out parameters that are set by the chart default values already (like image tags, etc), and rather concentrate on the parameters that are important to be configured (and maybe adding a comment for them what they are used for). 
Compare to the example for the access node: https://github.com/DOME-Marketplace/integration-guide/blob/iam-minimal-config/config/accessnode.yaml

Reply:
I'll update it too, thanks